README.md

```diff
@@ -1,4 +1,23 @@
-# mix
+# Mix
 
 [![Join the chat at https://gitter.im/ethereum/mix](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ethereum/mix?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 The Mix Ethereum Dapp Development Tool
+
+Mix is an IDE that allows developers to build, deploy contracts and decentralized applications on top of the Ethereum blockchain.
+
+![Mix Screenshot](https://github.com/yann300/mix/blob/ReadMe/MixScreenshot.png?raw=true)
```

Comment:
Could you make the window in the screenshot a bit smaller so that the relative font size is increased?
Also please link an image in the official repository, not your personal one (you can use relative links).